### PR TITLE
Windows kubeproxy flags cleanup

### PR DESCRIPTION
**Pull request by mtaufen**
**What type of PR is this?**
/kind cleanup
/priority important-soon
/sig node
/assign @pjh @yujuhong 

**What this PR does / why we need it**:
Follow-up to https://github.com/kubernetes/kubernetes/pull/74736

- Pass most of the flags via kube-env, similar to Kubelet, to make it
  easier to reconfigure.
- Use kube-proxy's kubeconfig file to pass the API server address,
  rather than the --master flag.
- Make KUBEPROXY_ARGS a required variable in Windows kube-env, similar
  to KUBELET_ARGS.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by mtaufen** [comment]
Burned by a trailing comma. Everything works now that it is removed. Re-adding label.

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -572,7 +573,8 @@ contexts:
   name: service-account-context
 current-context: service-account-context'.`
     replace('KUBEPROXY_TOKEN', ${kube_env}['KUBE_PROXY_TOKEN']).`
-    replace('CA_CERT', ${kube_env}['CA_CERT'])
+    replace('CA_CERT', ${kube_env}['CA_CERT']).`
+    replace('APISERVER_ADDRESS', ${apiserverAddress})
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: apiserverAddress is not set here - this should be ${kube_env}['KUBERNETES_MASTER_NAME'] ?

Comment (mtaufen) [reply cluster/gce/windows/k8s-node-setup.psm1]: Ah you're right! Create-KubeletKubeconfig sets apiserverAddress, but not Create-KubeproxyKubeconfig!

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 2]
```diff
@@ -572,7 +573,8 @@ contexts:
   name: service-account-context
 current-context: service-account-context'.`
     replace('KUBEPROXY_TOKEN', ${kube_env}['KUBE_PROXY_TOKEN']).`
-    replace('CA_CERT', ${kube_env}['CA_CERT'])
+    replace('CA_CERT', ${kube_env}['CA_CERT']).`
+    replace('APISERVER_ADDRESS', ${kube_env}['KUBERNETES_MASTER_NAME'])
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1]: Add KUBERNETES_MASTER_NAME to the list of required kube_env keys in the comment for this function.

**Review by pjh — APPROVED** [review]

**Comment by mtaufen** [comment]
smoke test passes on latest

**Pull request merged**